app/controllers/homes_controller.rb

```diff
@@ -3,6 +3,7 @@ def show
     if signed_out?
       render "pages/landing"
     else
+      @recommended_items = RecommendedItem.all
```

Comment:
Sounds like a good problem to have! I'd prefer to keep it as-is for now since we'll probably paginate later (with something like `will_paginate`) and the method for testing that will be different (and easier) than testing for a simple `.limit(50)`. For one thing, we won't have to create 50 records.